frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -65,8 +65,13 @@ function EmbeddingOption({
   );
 }
 
-export const StaticEmbeddingOptionCard = () => {
-  const enabled = useSetting("enable-embedding");
+interface EmbeddingOptionCardProps {
+  onToggle: ChangeEventHandler<HTMLInputElement>;
+}
+export const StaticEmbeddingOptionCard = ({
+  onToggle,
+}: EmbeddingOptionCardProps) => {
+  const isStaticEmbeddingEnabled = useSetting("enable-embedding-static");
```

Comment:
most of the `enabled` variable has been renamed to their respective setting name.